docs/src/IceNucleation.md

```diff
@@ -42,3 +42,103 @@ Both parameters are dependent on aerosol properties and temperature.
     For a ``f_i`` values above 0.08 or ``S_i`` between 1.35 and 1.5,
     freezing occurs in a different ice nucleation mode
     (either a second deposition or other condensation type mode).
+
+## ABIFM for Sulphuric Acid Containing Droplets
+Water Activity-Based Immersion Freezing Model (ABFIM) is a method of parameterizing immersion freezing inspired by the time-dependent classical nucleation theory. The nucleation rate coefficient, ``J``, can be determined by the water activity, ``a_w``. This parameterization follows [KnopfAlpert2013](@cite), [Koop2002](@cite), [MurphyKoop2005](@cite), and [Luo1995](@cite). In this model, aerosols are assumed to contain an insoluble and soluble material. When immersed in water, the soluble material diffuses into the liquid water to create a sulphuric acid solution.
+
+Using empirical coefficients, ``m`` and ``c``, from [KnopfAlpert2013](@cite), the heterogeneous nucleation rate coefficient in units of ``cm^{-2}s^{-1}`` can be determined by the linear equation
```

Comment:
Maybe we can use the same strategy as Nat was with aerosol nucleation. I.e. write the rates in the docs as they are in the original papers. But then mention in the docs that the actual source code is in base SI units (so meters and not cm)?